sentry/resource_sentry_key.go

```diff
@@ -149,6 +163,11 @@ func resourceSentryKeyUpdate(d *schema.ResourceData, meta interface{}) error {
 		Name: d.Get("name").(string),
 	}
 
+	rateLimit := getKeyRateLimit(d.Get("rate_limit_count").(int), d.Get("rate_limit_window").(int))
```

Comment:
I think no separate function needed for this logic, it could be inlined for more readability.

```golang
	rlc := d.Get("rate_limit_count").(int)
	rlw := d.Get("rate_limit_window").(int)
	if rlc + rlw > 0 {
		params.RateLimit = &sentryclient.ProjectKeyRateLimit{
			Window: rlw,
			Count:  rlc,
		}
	}
```

Please use concise naming instead of `fullDescriptiveCamelCaseNames`. It's a best practis and an ideom for Golang.

Reply:
Oh cool! Didn't know that